vmas/scenarios/navigation.py

```diff
@@ -283,6 +285,99 @@ def extra_render(self, env_index: int = 0) -> "List[Geom]":
         return geoms
 
 
+class HeuristicPolicy(BaseHeuristicPolicy):
+    def __init__(self, ctrl_params=[0.2, 1.0, 1.0, 100.0], *args, **kwargs):
```

Comment:
Here could we have `clf_epsilon=0.2, cbf_eta=1.0` etc. ?

Reply:
The clf_epsilon defines the convergence rate of the Lyapunov function. Let me remove cbf parameters for now because it's not added for now.

Reply:
I just meant having each parameter separate instead of a list.
So that users can change just one.

Reply:
I see. One moment, let me modify it.